docs/changelog.rst

```diff
@@ -11,6 +11,9 @@ KSQL 5.2 includes new features, including:
 * Added support for read-after-write consistency: new commands don't execute until previous commands have finished executing.
   This feature is enabled by default in the CLI (`#2280 <https://github.com/confluentinc/ksql/pull/2280>`_)
   and can be implemented by the user for the REST API (:ref:`coordinate_multiple_requests`).
+* A log of record processing events to help users debug their KSQL queries. The log can be configured
```

Comment:
nit: the other bullets start with "Added" and breaking parallelism reads a little harshly. Either update to start with "Added" or update the others to match yours?